typescript/infra/scripts/funding/fund-relayers-from-deployer.ts

```diff
@@ -234,20 +240,52 @@ function submitFunderBalanceMetrics(balances: FunderBalance[]) {
       .set(balance);
   }
 
-  gateway
-    .push({ jobName: 'relayer_funder' })
-    .then(({ resp, body }) => {
-      const statusCode =
-        typeof resp == 'object' && resp != null && 'statusCode' in resp
-          ? (resp as any).statusCode
-          : 'unknown';
-      console.debug(
-        `Prometheus push resulted with status ${statusCode} and body ${body}`,
-      );
-    })
-    .catch((err) => {
-      console.error(`Error pushing metrics: ${err}`);
-    });
+  const { resp, body } = await gateway.push({ jobName: 'relayer_funder' });
+  const statusCode =
+    typeof resp == 'object' && resp != null && 'statusCode' in resp
+      ? (resp as any).statusCode
+      : 'unknown';
+  log(`Prometheus metrics pushed to PushGateway`, {
+    statusCode,
+    body,
+  });
+}
+
+function log(message: string, data?: any) {
+  logWithFunction(console.log, message, data);
```

Comment:
Did you test this, afraid you might need to bind.

Reply:
I did :)
```
$ ts-node ./scripts/funding/fund-relayers-from-deployer.ts -e testnet2
{"chain":"alfajores","funder":{"address":"0xfaD1C94469700833717Fa8a3017278BC1cA8031C","balance":"4.018313295","desiredRelayerBalance":"1"},"message":"Funding relayers on chain..."}
{"relayer":{"address":"0xe62512d3c303e0540ed0338da3f43d8f7b0a369e","identifier":"alias/abacus-testnet2-key-kovan-relayer","chain":"kovan"},"balance":"0.9899836205","message":"Relayer balance"}
{"relayer":{"address":"0xa9521dc08c963796f615817a20b2b2981def8bb5","identifier":"alias/abacus-testnet2-key-fuji-relayer","chain":"fuji"},"balance":"0.990055339","message":"Relayer balance"}
...
```